cms/static/sass/views/_certificates.scss

```diff
@@ -125,15 +125,15 @@
 
       .actions {
         @include transition(opacity .15s .25s ease-in-out);
```

Comment:
ah yeah I think you are correct. I'll note this in the parent PR to keep this one focused on just the RTL work but it is a good catch!